temba/channels/views.py

```diff
@@ -1920,7 +1920,7 @@ def get_context_data(self, **kwargs):
             context["configuration_urls"] = channel_type.get_configuration_urls(self.object)
             context["show_public_addresses"] = channel_type.show_public_addresses
 
-            if hasattr(settings, 'SUB_DIR'):
+            if settings.SUB_DIR:
```

Comment:
This has bugged me for a bit too. Your solution is still vulnerable to a `NameError: name 'SUB_DIR' is not defined` if someone loads a settings file from RP which doesn't contain a trace of `SUB_DIR`. I recommend maintaining both tests.
```suggestion
            if hasattr(settings, 'SUB_DIR') && settings.SUB_DIR:
```